tasks/internal_reconcile_report_phantom/schemas/input.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/internal_reconcile_report_phantom/schemas/input.json",
+    "title": "internal_reconcile_report_phantom Lambda input",
+    "description": "The input for the internal_reconcile_report_phantom Lambda.", 
+    "type": "object",
+    "required": [
+      "reportId", "pageNumber"
+    ],
+    "properties": {
+      "reportId": {
```

Comment:
We should stay consistent... Currently we use `jobId` for other lambdas and internally in the DB.

Reply:
so replace`reportId` with `jobId`?

Reply:
That'd be my vote.

Reply:
changed